### DISCUSSION TITLE
1.4.0 bug reports

### DISCUSSION THREAD

**Discussion by eschutho** [apache/superset#18119] [General]
We will be releasing continued patch releases on this minor bump, and creating a space here for any PRs or issues for fixes or bugs found  to be listed here for consideration into future patch releases. 

Current list of PRs for 1.4.1:
https://github.com/apache/superset/pull/16869
https://github.com/apache/superset/pull/18048
https://github.com/apache/superset/pull/17707 (This is in 1.4.0 but a subsequent cherry overwrote some necessary changes)
https://github.com/apache/superset/pull/17992
https://github.com/apache/superset/pull/17961
https://github.com/apache/superset/pull/17573
https://github.com/apache/superset/pull/18006
https://github.com/apache/superset/pull/17981
https://github.com/apache/superset/pull/17984
https://github.com/apache/superset/pull/17980

------------
Current list of PRs for 1.4.2
https://github.com/apache/superset/pull/17974
https://github.com/apache/superset/pull/17726
https://github.com/apache/superset/discussions/18187
https://github.com/apache/superset/commit/6e5392db1175039698859804554bf81532252de6
https://github.com/apache/superset/issues/18220

-------
Open Issues (under consideration if we can get a fix):
https://github.com/apache/superset/issues/16975
https://github.com/apache/superset/issues/18085
https://github.com/apache/superset/issues/18129
https://github.com/apache/superset/issues/18109
https://github.com/apache/superset/issues/12302 (it's a little unclear whether this fix will be a feature or bug fix)

**Comment by kamalkeshavani-aiinside** [comment]
Here are my suggestions:
- apache/superset#17992
- apache/superset#17982
- apache/superset#17961

related fixes:
- apache/superset#17573
- apache/superset#18006

**Reply by eschutho** [reply]
Thank you @kamalkeshavani-aiinside. I have added these to the list. 👍

**Comment by villebro** [comment]
FYI @rumbin tagging this issue here if someone has time to fix this before the cut: #16975

**Comment by rumbin** [comment]
Not sure if this can be fixed for 1.4.1: #18085

**Reply by eschutho** [reply]
I added it to the "open issues" list.

**Comment by rumbin** [comment]
IMHO the broken area chart belongs in this list, as it is so user-facing and existing charts/dashboards are affected: #18129

**Comment by rafalpas** [comment]
Please consider https://github.com/apache/superset/issues/18109 for 1.4.1.

**Reply by eschutho** [reply]
Thank you. Added to open issues.
